Assets/NuGet/Editor/NugetHelper.cs

```diff
@@ -208,6 +211,21 @@ private static void RunNugetProcess(string arguments, bool logOuput = true)
 
             LogVerbose("Running: {0} \nArgs: {1}", files[0], arguments);
 
+#if UNITY_EDITOR_OSX
+            var monoFile = "/Library/Frameworks/Mono.framework/Versions/Current/Commands/mono";
+            //TODO:using unity mono to pack the Nuget package.
+            var commandLine = " " + files[0] + " " + arguments;
```

Comment:
Seems like more of this code could be shared between the two platforms. Looks like only the `FileName` and `Arguments` are different?

Reply:
Done

Reply:
Yay!